### DISCUSSION TITLE
Notepad Type Block On Station Panel

### DISCUSSION THREAD

**Discussion by Countryman98** [AzuraCast/AzuraCast#5102] [Ideas]
How bout adding the ability for some sort of notepad 'block' on the top of the main station page somewhere where multiple admins can leave messages related to Server settings, Do's Dont's Etc to be brought to the attention of "Other" admins when they login?

Just an idea, Anyone else reckon it could be a handy feature?

**Comment by ItsMitchh** [comment]
Sounds possibly useful. Go ahead and post the request to https://azuracast.featureupvote.com
